webservices/tasks/download.py

```diff
@@ -25,6 +25,14 @@
 
 IGNORE_FIELDS = {'page', 'per_page', 'sort', 'sort_hide_null'}
 RESOURCE_WHITELIST = {
+    aggregates.ScheduleABySizeView,
+    aggregates.ScheduleAByStateView,
+    aggregates.ScheduleAByZipView,
+    aggregates.ScheduleAByEmployerView,
+    aggregates.ScheduleAByOccupationView,
+    aggregates.ScheduleBByRecipientView,
+    aggregates.ScheduleBByRecipientIDView,
+    aggregates.ScheduleBByPurposeView,
```

Comment:
After looking through this some, this is far too much work to try and make happen for the mini-release we're trying to get out the door.  We'll make a separate PR for it later.